### DISCUSSION TITLE
"Deprecated" doesn't mean "removed"

### DISCUSSION THREAD

**Discussion by martys71** [PowerDNS/pdns#14508] [General]
My PowerDNS server crashed yesterday after an upgrade. It turned out that if one has "master" or "slave" options in their config file the server will crash with the "unknown config option" fatal error. Isn't it a bit too harsh? Anyway in the documentanion both "master" and "slave" config options are marked as "deprecated", not as "removed" or "forbidden".

**Comment by Habbie** [comment]
https://doc.powerdns.com/authoritative/upgrading.html#to-4-9-0

I'll update settings.html, thanks for the notice
